.github/workflows/integration-tests.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         run: php ./tempest discovery:status
 
       - name: Execute tests
-        run: php -d"error_reporting = E_ALL & ~E_DEPRECATED" vendor/bin/phpunit --coverage-clover build/reports/clover.xml
+        run: php -d"error_reporting = E_ALL & ~E_DEPRECATED" -dmemory_limit=1G vendor/bin/phpunit --coverage-clover build/reports/clover.xml
 
       # Only upload coverage once.
       - if: matrix.php == '8.3' && matrix.database == 'sqlite' && matrix.os == 'ubuntu-latest' && matrix.stability == 'prefer-stable'
```

Comment:
BTW, does our coverage still work? We do a check here for 8.3, but we only run on 8.4

Reply:
I updated it for PHP 8.4!